docs/source/app-dev/bindings-ts/testing.rst

```diff
@@ -4,7 +4,7 @@
 Testing Your App
```

Comment:
I think we should rephrase the title a bit. This is really not about general daml testing, it’s very specifically about e2e UI testing for web frontends. Maybe "E2e UI testing of Daml Applications"?

Reply:
I was assuming the context made it clear (this is under the "JavaScript Client Libraries" header), but happy to change if you don't think context is clear enough:

<img width="173" alt="Screenshot 2021-08-04 at 14 58" src="https://user-images.githubusercontent.com/45561385/128184783-db337126-87f1-4681-b51a-2b1b4211fec7.png">

Reply:
Maybe "Testing Your Web App"?

Reply:
I like testing your web app :+1: